starter-code/public/javascript/map.js

```diff
@@ -0,0 +1,106 @@
+function startMap() {
+
+  var ironhackParis = {
+    lat: 48.8745686,
+    lng: 2.335433
+  };
+
+  var map = new google.maps.Map(
+    document.getElementById('map'), {
+      zoom: 15,
+      center: ironhackParis
+    }
+  );
+
+  var input = document.getElementById('beach-search');
+
+  var searchBox = new google.maps.places.SearchBox(input);
+
+  //create markers
+  var markers = [];
+
+  searchBox.addListener('places_changed', () => {
```

Comment:
This event is triggered everytime the content of the search box changes. I don't think you want to add a listener to it; you'd rather handle click event on the *Find the beach* button.